conftest.py

```diff
@@ -20,10 +20,16 @@
 import os
 import shutil
 import tempfile
+import warnings
 
+from keras.preprocessing.image import ImageDataGenerator
+from mxnet import gluon
```

Comment:
Please don't use framework imports at the top to allow running single tests without having to install all frameworks, follow the example on `dev_1.4.0`.

Reply:
ok yes that's a good point. so should I move all these imports in each method? Actually, that's one thing I had in my mind for a while (not for this PR of course but in the longer term). To deal with this issue, should we not perhaps split the test/util.py file into framework specific test/util_pytorch etc? This way we wouldn't need to place these imports in multiple methods and would enable users to have to install only 1 framework?

Reply:
I'm not sure of it is required at this stage, for 1.4.0 we have already fixed `conftest.py` and this PR would reintroduce these imports at the top, therefore I would suggest to follow the example on `dev_1.4.0` for now.